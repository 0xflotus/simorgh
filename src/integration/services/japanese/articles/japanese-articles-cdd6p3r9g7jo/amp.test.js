/**
 * @service japanese
 * @pathname /japanese/articles/cdd6p3r9g7jo
 */

import runAmpTests from '../../../../pages/articles/ampTests';

describe('AMP japanese articles', runAmpTests);
