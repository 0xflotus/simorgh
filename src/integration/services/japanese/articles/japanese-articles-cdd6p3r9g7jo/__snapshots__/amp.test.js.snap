// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AMP japanese articles A11y I can see the skip to content link 1`] = `"コンテンツへ移動"`;

exports[`AMP japanese articles AMP Consent config 1`] = `
Object {
  "consents": Object {
    "user-consent": Object {
      "promptIfUnknownForGeoGroup": "eea",
      "promptUI": "consent-prompt",
    },
  },
  "policy": Object {
    "default": Object {
      "timeout": Object {
        "fallbackAction": "reject",
        "seconds": 0,
      },
      "waitFor": Object {
        "user-consent": Array [],
      },
    },
  },
}
`;

exports[`AMP japanese articles AMP Geo config 1`] = `
Object {
  "ISOCountryGroups": Object {
    "eea": Array [
      "at",
      "ax",
      "be",
      "bg",
      "bl",
      "cy",
      "cz",
      "de",
      "dk",
      "ea",
      "ee",
      "es",
      "fi",
      "fr",
      "gb",
      "gf",
      "gg",
      "gi",
      "gp",
      "gr",
      "hr",
      "hu",
      "ic",
      "ie",
      "im",
      "is",
      "it",
      "je",
      "li",
      "lt",
      "lu",
      "lv",
      "mf",
      "mq",
      "mt",
      "nc",
      "nl",
      "no",
      "pf",
      "pl",
      "pm",
      "pt",
      "re",
      "ro",
      "se",
      "si",
      "sj",
      "sk",
      "tf",
      "va",
      "wf",
      "yt",
    ],
  },
}
`;

exports[`AMP japanese articles Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/japanese"`;

exports[`AMP japanese articles Footer I can see the footer copyright and external linking text 1`] = `"© 2020 BBC.　BBCは外部サイトの内容に責任を負いません。 本サイトの外部リンク方針はこちら"`;

exports[`AMP japanese articles Header I can see the branding 1`] = `"BBC News, ニュース"`;

exports[`AMP japanese articles I can see an image with a caption 1`] = `"https://ichef.bbci.co.uk/news/640/cpsdevpb/cba9/test/c8130910-1109-11ea-acaa-cd607a38aea4.jpg"`;

exports[`AMP japanese articles I can see an image with a caption 2`] = `"画像説明, このテスト画像、著作権は、フランスの地図を示しています。画像は最初の3ブロックにあり、このキャプションがあります。"`;

exports[`AMP japanese articles SEO Canonical link 1`] = `"http://localhost:7080/japanese/articles/cdd6p3r9g7jo"`;

exports[`AMP japanese articles SEO Dir attribute 1`] = `"ltr"`;

exports[`AMP japanese articles SEO FB admins 1`] = `"100004154058350"`;

exports[`AMP japanese articles SEO FB app ID 1`] = `"1609039196070050"`;

exports[`AMP japanese articles SEO Heading level 1 1`] = `"これは、このテスト記事の見出しです"`;

exports[`AMP japanese articles SEO Lang attribute 1`] = `"ja"`;

exports[`AMP japanese articles SEO Linked data 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "Article",
  "author": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/include/articles/public/japanese/images/metadata/poster-1024x576.png",
      "width": 1024,
    },
    "name": "BBCニュース",
    "noBylinesPolicy": "https://www.bbc.com/japanese/help-49677253#authorexpertise",
  },
  "dateModified": "2019-11-27T11:38:13.250Z",
  "datePublished": "2019-11-27T11:38:13.250Z",
  "headline": "これは、このテスト記事の見出しです",
  "image": Object {
    "@type": "ImageObject",
    "height": 576,
    "url": "https://news.files.bbci.co.uk/include/articles/public/japanese/images/metadata/poster-1024x576.png",
    "width": 1024,
  },
  "mainEntityOfPage": Object {
    "@id": "http://localhost:7080/japanese/articles/cdd6p3r9g7jo",
    "@type": "WebPage",
    "name": "これは、このテスト記事の見出しです",
  },
  "publisher": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/include/articles/public/japanese/images/metadata/poster-1024x576.png",
      "width": 1024,
    },
    "name": "BBCニュース",
    "publishingPrinciples": "https://www.bbc.com/japanese/help-49677253",
  },
  "thumbnailUrl": "https://news.files.bbci.co.uk/include/articles/public/japanese/images/metadata/poster-1024x576.png",
  "url": "http://localhost:7080/japanese/articles/cdd6p3r9g7jo",
}
`;

exports[`AMP japanese articles SEO OG description 1`] = `"これは、このテスト記事の見出しです"`;

exports[`AMP japanese articles SEO OG image 1`] = `"https://news.files.bbci.co.uk/include/articles/public/japanese/images/metadata/poster-1024x576.png"`;

exports[`AMP japanese articles SEO OG image alt 1`] = `"BBCニュース"`;

exports[`AMP japanese articles SEO OG locale 1`] = `"ja-JP"`;

exports[`AMP japanese articles SEO OG site name 1`] = `"BBCニュース"`;

exports[`AMP japanese articles SEO OG title 1`] = `"これは、このテスト記事の見出しです - BBCニュース"`;

exports[`AMP japanese articles SEO OG type 1`] = `"article"`;

exports[`AMP japanese articles SEO OG url 1`] = `"http://localhost:7080/japanese/articles/cdd6p3r9g7jo"`;

exports[`AMP japanese articles SEO Page title 1`] = `"これは、このテスト記事の見出しです - BBCニュース"`;

exports[`AMP japanese articles SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`AMP japanese articles SEO Twitter card 1`] = `"summary_large_image"`;

exports[`AMP japanese articles SEO Twitter creator 1`] = `"@bbcnewsjapan"`;

exports[`AMP japanese articles SEO Twitter description 1`] = `"これは、このテスト記事の見出しです"`;

exports[`AMP japanese articles SEO Twitter image alt 1`] = `"BBCニュース"`;

exports[`AMP japanese articles SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/include/articles/public/japanese/images/metadata/poster-1024x576.png"`;

exports[`AMP japanese articles SEO Twitter site 1`] = `"@bbcnewsjapan"`;

exports[`AMP japanese articles SEO Twitter title 1`] = `"これは、このテスト記事の見出しです - BBCニュース"`;
